.github/workflows/build-and-test.yml

```diff
@@ -33,9 +33,7 @@ jobs:
     - name: cmake
       uses: lukka/run-cmake@v10
       with:
-        configurePreset: "ci-${{matrix.os}}"
-        buildPreset: "ci-${{matrix.os}}"
-        testPreset: "ci-${{matrix.os}}"
+        workflowPreset: "ci-${{matrix.os}}"
```

Comment:
Config change without version change?

Reply:
Not sure what you mean

Reply:
Whether workflowPreset has always existed in that version of the action

Reply:
I believe so? Still not quite sure what you mean by the "version" of the action. Do you mean the commit?

Reply:
The
> uses: lukka/run-cmake@v10
specifically the "v10" git tag of that repo

Reply:
Ah. The current v10(.7) has it. The original v10(.0) did not.